### ISSUE TITLE
Dpad stops working rog ally

### ISSUE THREAD

**Issue by ripplingsnake** [ShadowBlip/HandyGCCS#136]
Just recently the dpad has stopped responding but all other buttons are fine

**Comment by pastaq** [comment]
Have you ruled out a hardware issue?

**Comment by ripplingsnake** [comment]
Yes I have reinstalled handy on arch based distro and enabled the service but the  buttons do not work . I want to use handy rather than say hhd as there is to much clutter on it now imo

**Comment by pastaq** [comment]
What kernel are you running?
I'm not able to reproduce this in ChimeraOS. You may be missing some patches in the kernel.
